packages/one-app-bundler/__tests__/webpack/module/webpack.server.spec.js

```diff
@@ -23,4 +23,9 @@ const { validateWebpackConfig } = require('../../../test-utils');
 
 describe('webpack/module.server', () => {
   it('should export valid webpack config', () => validateWebpackConfig(webpackConfig));
+
+  it('should define global.BROWSER to be false', () => {
+    expect(webpackConfig).toHaveProperty('plugins', expect.any(Array));
+    expect(webpackConfig.plugins).toContainEqual({ definitions: { 'global.BROWSER': 'false' } });
```

Comment:
considering all our `global.BROWSER` checks check for truthyness is this going to cause issues as `'false'` is actually truthy

Reply:
>Tip: Note that because the plugin does a direct text replacement, the value given to it must include actual quotes inside of the string itself. Typically, this is done either with alternate quotes, such as '"production"', or by using JSON.stringify('production').

https://webpack.js.org/plugins/define-plugin/

Working with a static file rather than runtime memory, so the definition has to be a string but execution will evaluation what is in the string. Got me when we first added `global.BROWSER` 😅